benchmarks/multiplatform/benchmarks/build.gradle.kts

```diff
@@ -142,6 +158,41 @@ tasks.register("buildD8Distribution", Zip::class.java) {
     destinationDirectory.set(rootProject.layout.buildDirectory.dir("distributions"))
 }
 
+tasks.register("runBrowserAndSaveStats") {
+    fun printProcessOutput(inputStream: java.io.InputStream) {
+        Thread {
+            inputStream.bufferedReader().use { reader ->
+                reader.lines().forEach { line ->
+                    println(line)
+                }
+            }
+        }.start()
+    }
+
+    doFirst {
+        var serverProcess: Process? = null
+        var clientProcess: Process? = null
+        try {
+            serverProcess = ProcessBuilder("./gradlew", "benchmarks:run", "-PrunArguments=runServer=true saveStatsToJSON=true")
+                .start()
+            printProcessOutput(serverProcess.inputStream)
+            printProcessOutput(serverProcess.errorStream)
+
+            clientProcess = ProcessBuilder("./gradlew", "benchmarks:wasmJsBrowserProductionRun")
```

Comment:
looks like `saveStatsToJSON=true` is not passed here, the browser doesn't make `/benchmark` requests

Reply:
fixed